.github/workflows/go.yml

```diff
@@ -45,4 +45,5 @@ jobs:
           version: "v0.7.0"
 
       - name: Run e2e tests
-        run: make go-integration
+        run: make e2e-test
```

Comment:
With this renaming, should we change the name of this directory `https://github.com/crossplane/oam-kubernetes-runtime/tree/master/test/integration` ?